### PR TITLE
feat(ui): add new page to display real-time events

### PR DESCRIPTION
**Pull request by wwayne**
### Notable changes
* Removed `@/components/ui/date-range-picker`, which was copied from shadcn before. We are not using it anymore.
* Created a new component with the same name `@/components/date-range-picker`. The component is used in Reports and Activities pages.

### Screen record
https://jam.dev/c/c79f8e42-df8f-47d5-a119-c8d854f7194b

fix TAB-582

### CODE REVIEW

File: ee/tabby-ui/app/(dashboard)/activity/page.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Metadata } from 'next'
+
+import Activity from './components/activity'
+
+export const metadata: Metadata = {
+  title: 'Activity'
```
Comment (wsxiaoys) [inline ee/tabby-ui/app/(dashboard)/activity/page.tsx:6]:
```suggestion
  title: 'Activities'
```

File: ee/tabby-ui/app/(dashboard)/components/sidebar.tsx [hunk 2]
```diff
@@ -68,6 +68,7 @@ export default function Sidebar({ children, className }: SidebarProps) {
                     <SidebarButton href="/cluster">Cluster</SidebarButton>
                     <SidebarButton href="/jobs">Jobs</SidebarButton>
                     <SidebarButton href="/reports">Reports</SidebarButton>
+                    <SidebarButton href="/activity">Activity</SidebarButton>
```
Comment (wsxiaoys) [inline ee/tabby-ui/app/(dashboard)/components/sidebar.tsx:71]:
```suggestion
                    <SidebarButton href="/activity">Activities</SidebarButton>
```

File: ee/tabby-ui/app/(dashboard)/activities/components/activity.tsx [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+                <div className="flex w-[100px] items-center justify-center text-sm font-medium">
+                  Page {page}
+                </div>
+                <div className="flex items-center space-x-2">
+                  <Button
+                    variant="outline"
+                    className="h-8 w-8 p-0"
+                    disabled={!data?.userEvents.pageInfo?.hasNextPage}
+                    onClick={e => {
+                      setQueryVariables({
```
Comment (liangfung) [inline ee/tabby-ui/app/(dashboard)/activities/components/activity.tsx:203]: I recommend using relayPagination(), let's discuss offline

File: ee/tabby-ui/app/(dashboard)/activities/components/activity.tsx [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+}
+
+function ActivityRow({
+  activity
+}: {
+  activity: ListUserEventsQuery['userEvents']['edges'][0]['node']
+}) {
+  const { theme } = useTheme()
+  const [members] = useAllMembers()
+  const [isCollapse, setIsCollapse] = React.useState(false)
```
Comment (liangfung) [inline ee/tabby-ui/app/(dashboard)/activities/components/activity.tsx:243]: If the default value is false, semantically, should it be `isExpanded`?

File: ee/tabby-ui/app/(dashboard)/activities/components/activity.tsx [hunk 5]
```diff
@@ -0,0 +1,333 @@
...
+                    </Table>
+                  </CardContent>
+                </>
+              )}
+            </Card>
+
+            {(data?.userEvents.pageInfo?.hasNextPage ||
+              data?.userEvents.pageInfo?.hasPreviousPage) && (
+              <div className="flex justify-end">
+                <div className="flex w-[100px] items-center justify-center text-sm font-medium">
```
[the remark this answers is not in the document]
Comment (wwayne) [reply ee/tabby-ui/app/(dashboard)/activities/components/activity.tsx:230]: After the discussion, we decided to keep using the current one to maintain the style.

File: ee/tabby-ui/app/(dashboard)/activities/components/activity.tsx (hunk 3 above)
Comment (wwayne) [reply ee/tabby-ui/app/(dashboard)/activities/components/activity.tsx:240]: Checked with Jueliang, relayPagination seems not suitable for this case

File: ee/tabby-ui/app/(dashboard)/activities/components/activity.tsx (hunk 4 above)
Comment (wwayne) [reply ee/tabby-ui/app/(dashboard)/activities/components/activity.tsx]: updated to using isExpanded

**Pull request merged**